addons/acre2/config.cpp

```diff
@@ -22,3 +22,24 @@ class cfgPatches
 /*#include "CfgLoadouts.hpp"*/
 
 #include "display3DEN.hpp"
+
+class acre_sys_spectator_RscRadios;
+class acre_sys_spectator_RscSpeaking;
+class RscControlsGroupNoScrollbars;
+
+// Extend spectator
+class EGVAR(spectator,dialog) {
+    class controls {
+        class TargetUnitInfo: RscControlsGroupNoScrollbars {
+            idc = SPECTATOR_IDC_RADIO_INFO;
+            h = "((7.1) * ((((safezoneW / safezoneH) min 1.2)/ 1.2)/ 25))";
```

Comment:
`h = 7.1 * GUI_GRID_H;`

Reply:
This is the initial integration which will give us an opportunity to try this out, I think once we know how useful it is the final placement will need some work which can be the scope of a later PR.

Reply:
No doubt, just that is easier to work with macros instead of the full safezone statement. GUI_GRID is identical.